src/view.tree.js

```diff
@@ -1,5 +1,6 @@
 class TreeView {
   constructor($dom, store, adapter) {
+    const isEdge = window.navigator.userAgent.indexOf('Edge') > 0;
```

Comment:
Would be great if there's a way to make it work without specific browser detection & handling like this.

Reply:
It seems that this problem is resolved in the latest version of Edge. I have removed the browser detection code.